pages.es/common/kill.md

```diff
@@ -0,0 +1,37 @@
+# kill
+
+> Este comando es utilizado para matar/terminar procesos.
+> kill envia una señal para terminar uno o un grupo de procesos.
+> Mas Informacion: <https://linuxhint.com/linux-kill-command/>.
+
+- Mata un proceso o grupo de procesos.
+
+`kill signal PID`
+
+- Lista de las señales disponibles (para ser utilizadas sin el prefijo SIG):
+
+`kill -l`
+
+- Terminar un trabajo en segundo plano:
+
+`kill %{{job_id}}`
+
+- Termina un programa con  SIGHUP (hang up/colgar). Varios daemons se reiniciaran en lugar de terminar:
+
+`kill -{{1|HUP}} {{process_id}}`
+
+- Terminar un programa mediante SIGINT (interrupt/interrumpir). Esto normalmente lo hace el usuario presionando Ctrl + C:
+
+`kill -{{2|INT}} {{process_id}}`
+
+- Señal del sistema que termina de inmediato un programa (Sin oportunidad de captar la señal):
```

Comment:
Here `signal` is used as verb.
```suggestion
- Señala al sistema operativo para terminar inmediatamente un programa (el cual no tiene oportunidad de capturar la señal):
```
`capturar` is better than `captar` because if you capture a signal the OS can't receive it.